_posts/2021-05-26-registry-security.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Securing Apicurio Registry 2.0 using Keycloak"
+date:   2021-05-28 12:00:00
+author: carles
+categories: blog registry security
+---
+
+
+Hey everyone, in Apicurio Registry 2.0.0.Final we've introduced support for some security concepts. In this blog post, I will try to explain them and show you how to configure them.
+
+Core Features
+===
+Essentially in this blog post, we will cover the following:
+
+* **Authentication based on Keycloak** - optionally protect the registry such that the API requires users to authenticate (BASIC and OAuth supported)
+* **Role based authorization** - when authentication is enabled, you can also enable this feature, and users will need to have at least one role of (sr-admin, sr-developer, sr-readonly)
```

Comment:
Suggest adding hyphen in "Role-based authorization"

Suggest "...at least one role of `sr-admin`, `sr-developer`, `sr-readonly` "